scripts/build/babel-plugins/strip-target-env.js

```diff
@@ -0,0 +1,36 @@
+"use strict";
+
+//
+// BEFORE:
+//   if (!process.env.PRETTIER_TARGET_UNIVERSAL) {
+//     foo();
+//   }
+//
+// AFTER:
+//   // in standalone mode: no output
+//   // in regular mode:
+//   foo();
+//
```

Comment:
This looks like the `NODE_ENV` replacing magic to me, we should be able to use the same magic (dead code elimination) in this case, and limit it to the `standalone.js` bundling.

https://github.com/prettier/prettier/blob/952bc0cc039c034c5fe651864db528dcd9800e9e/scripts/build/bundler.js#L97-L99

https://github.com/prettier/prettier/blob/952bc0cc039c034c5fe651864db528dcd9800e9e/src/doc/doc-builders.js#L15-L17

Reply:
Are those statements removed from the outputted file or does the if statement stay there?

Reply:
They'll be removed by the dead code elimination if it replaced correctly.

```js
// source
if (process.env.NODE_ENV) { doSomething() }

// input (replaced)
if (false) { doSomething() }

// output (bundled)
/* nothing */
```